### PR TITLE
CAM Terrafrom supports 2 additional providers

### PR DESCRIPTION
**Pull request by DenesPal**
@gsanchezu please review. Thanks.

### CODE REVIEW

File: Cloud Application Manager/Automating Deployments/template-box.md [hunk 1]
```diff
@@ -7,14 +7,14 @@
...
 
-* [Overview](#overview)
-* [Audience](#audience)
-* [Prerequisites](#prerequisites)
-* [Deploy Using CloudFormation Templates](#deploy-using-cloudformation-templates)
-* [Deploy Using Azure Resource Manager Templates](#deploy-using-azure-resource-manager-templates)
-* [Deploy Using Terraform Templates](#deploy-using-terraform-templates)
-* [Deploy Using Google Templates](#deploy-using-google-templates)
-* [Getting General Support](#getting-general-support)
+- [Overview](#overview)
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/template-box.md]: Please, keep the asterisks for the ToC bullet points

File: Cloud Application Manager/Automating Deployments/template-box.md [hunk 2]
```diff
@@ -59,13 +59,20 @@ For more information, go to [ARM templates](../Automating Deployments/arm-templa
...
+-  VMware vCenter/vSphere
+-  VMware vCloud Director
 
 This feature provides you with the ability to use the power of Cloud Application Manager (instance history, Lifecycle Editor, bindings, box versioning) combined with any service supported by Terraform.
 
-To learn more about:
+#### To learn more about Terraform
 
-In order to understand Terraform configurations see [here](https://www.terraform.io/docs/configuration/index.html).
+In order to understand Terraform's configurations language see [Terraform 0.11 documentation here](https://www.terraform.io/docs/configuration-0-11/index.html). Currently Cloud Application Manager can deploy **Terraform version 0.11** templates.
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/template-box.md]:
```suggestion
In order to understand Terraform's configurations language see [Terraform 0.11 documentation here](https://www.terraform.io/docs/configuration-0-11/index.html). Currently, Cloud Application Manager supports the deployment of **Terraform version 0.11** templates.
```

File: Cloud Application Manager/Automating Deployments/terraform-template-box.md [hunk 3]
```diff
@@ -7,17 +7,17 @@
...
 **In this article:**
 
-* [Overview](#overview)
-* [Audience](#audience)
-* [Prerequisites](#prerequisites)
-* [Create a Terraform Template and apply configuration](#create-a-terraform-template-and-apply-configuration)
-* [Update a Terraform configuration in Real-Time](#update-a-terraform-configuration-in-real-time)
-* [Connect to Other Cloud Application Manager Boxes over Bindings](#connect-to-other-cloud-application-manager-boxes-over-bindings)
-* [Getting General Support](#getting-general-support)
+- [Overview](#overview)
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/terraform-template-box.md]: Please, keep the asterisks for the ToC bullet points

File: Cloud Application Manager/Automating Deployments/terraform-template-box.md [hunk 4]
```diff
@@ -7,17 +7,17 @@
...
+- [Audience](#audience)
+- [Prerequisites](#prerequisites)
+- [Create a Terraform Template and apply configuration](#create-a-terraform-template-and-apply-configuration)
+- [Update a Terraform configuration in Real-Time](#update-a-terraform-configuration-in-real-time)
+- [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/terraform-template-box.md]: Why you changed this one? Now there are two supports, and no bindings section

File: Cloud Application Manager/Automating Deployments/terraform-template-box.md [hunk 5]
```diff
@@ -26,14 +26,23 @@ All users of Cloud Application Manager who wants to define and use Terraform Tem
 ### Prerequisites
 
 * An active *Cloud Application Manager* account
-* Having configured a provider of type [*AWS*](../Deploying Anywhere/using-your-aws-account.md), [Microsoft Azure](../Deploying Anywhere/using-microsoft-azure.md), [Google](../Deploying Anywhere/using-google-cloud.md) or [CenturyLink Cloud](../Deploying Anywhere/using-centurylink-cloud.md)
+* Having configured a provider with type of either
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/terraform-template-box.md]:
```suggestion
* Having configured a provider of either one of these types:
```

File: Cloud Application Manager/Automating Deployments/terraform-template-box.md [hunk 6]
```diff
@@ -26,14 +26,23 @@ All users of Cloud Application Manager who wants to define and use Terraform Tem
...
+  * [AWS](../Deploying Anywhere/using-your-aws-account.md)
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/terraform-template-box.md]: You can use "-" for second level bullet points

File: Cloud Application Manager/Automating Deployments/terraform-template-box.md [hunk 7]
```diff
@@ -26,14 +26,23 @@ All users of Cloud Application Manager who wants to define and use Terraform Tem
...
+  * [VMware vCenter/vSphere](../Deploying Anywhere/using-the-vmware-vcenter-private-datacenter.md)
+  * [VMware vCloud Director](../Deploying Anywhere/orchestrating-vcloud-air-vcloud-director-deployments.md)
 
 ### Create a Terraform Template and apply configuration
 
 The Terraform Template box consists mainly on a list of template files where you describe all the cloud resources you need to run your application. Cloud Application Manager parses the templates and automatically shows input parameters under a section called Variables. This enables you to customize a template easily.
 
 We use a sample Basic Two-Tier Architecture in Google Cloud with Terraform templates to show how to create and launch a Terraform box template in Cloud Application Manager.
 
+Cloud Application Manager supports Terraform configuration **version 0.11**.
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/terraform-template-box.md]:
```suggestion
Cloud Application Manager currently supports Terraform configuration **version 0.11**.
```

File: Cloud Application Manager/Automating Deployments/terraform-template-box.md [hunk 8]
```diff
@@ -90,12 +99,13 @@ We use a sample Basic Two-Tier Architecture in Google Cloud with Terraform templ
...
 
         * Variables available on Google: `${var.provider_google_region}` and `${var.provider_google_zone}`
         * Variables available on Amazon: `${var.provider_aws_region}`
 
-**Note:** As you’re authoring, it’s important to check that the template is valid. While Cloud Application Manager validates the correctness of format and the template syntax correctness, we can’t know whether resources specified are available in the provider used or whether property values of a resource are valid. For that level of checking, it’s best to test launch the Terraform box instance from Cloud Application Manager and refine the template in real-time.
+**Note:** As you’re authoring, it’s important to check that the template is valid. While Cloud Application Manager validates the correctness of format and the template syntax correctness, we can’t know whether resources specified are available in the provider used or whether property va
+lues of a resource are valid. For that level of checking, it’s best to test launch the Terraform box instance from Cloud Application Manager and refine the template in real-time.
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/terraform-template-box.md]: Remove this hard line wrap

File: Cloud Application Manager/Automating Deployments/terraform-template-box.md (hunk 4 above)
Comment (DenesPal) [reply Cloud Application Manager/Automating Deployments/terraform-template-box.md]: Hola Guillermo,

I use Visual Studio Code to edit Knowledge Base articles. The only reason I use it is that it has a TOC generation plugin. I regenerate the TOC every time I modify a document, to make sure it is accurate. (Now I've managed to set up TOC formatting correctly.)

After digging in the history, it seems that someone forgot to update TOC some time ago. There is no section about bindings in upstream/master.

-  Do you want me to remove the "Contaction Cloud Application Manager Support" section from the article?

- Which file serves as a standard example to copy headers, overview, audience, prerequisites and support sections from?

Thank you.

File: Cloud Application Manager/Automating Deployments/template-box.md [hunk 9]
```diff
@@ -7,14 +7,14 @@
 }}}
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/template-box.md]: Please, update metadata values (date, author)

File: Cloud Application Manager/Automating Deployments/terraform-template-box.md [hunk 10]
```diff
@@ -7,17 +7,17 @@
 }}}
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/terraform-template-box.md]: Please, update metadata values (date, author)

File: Cloud Application Manager/Automating Deployments/terraform-template-box.md (hunk 4 above)
Comment (gsanchezu) [reply Cloud Application Manager/Automating Deployments/terraform-template-box.md]: Hola @DenesPal,

Yes, you need to configure the plugin options to specify the ToC bullet points symbol to be used. There is another plugin that I use that is a sort of lint for MarkDown, which notifies you of some of these formatting issues.

Regarding the section, @EfrenRey may confirm if the bindings section was required (although it had no content, so it should be added), or if it was just added but not required). Additionally Denes, I saw that with your change there were two support sections, which I think that you have already fixed. Thanks. @EfrenRey, could you confirm if we need to document the bindings support for Terraform? Thanks!

Comment (DenesPal) [reply Cloud Application Manager/Automating Deployments/terraform-template-box.md]: I've removed old support section now.

File: Cloud Application Manager/Automating Deployments/terraform-template-box.md (hunk 10 above)
Comment (DenesPal) [reply Cloud Application Manager/Automating Deployments/terraform-template-box.md]: Date updated.
I don't want to take authorship 3 lines of text.

**Comment by gsanchezu** [comment]
Yeah, so for documentation you don't have to squash commits.

File: Cloud Application Manager/Automating Deployments/terraform-template-box.md [hunk 11]
```diff
@@ -1,23 +1,22 @@
...
 **In this article:**
 
-* [Overview](#overview)
-* [Audience](#audience)
-* [Prerequisites](#prerequisites)
-* [Create a Terraform Template and apply configuration](#create-a-terraform-template-and-apply-configuration)
-* [Update a Terraform configuration in Real-Time](#update-a-terraform-configuration-in-real-time)
-* [Connect to Other Cloud Application Manager Boxes over Bindings](#connect-to-other-cloud-application-manager-boxes-over-bindings)
-* [Getting General Support](#getting-general-support)
+* [Overview](#Overview)
```
Comment (gsanchezu) [inline Cloud Application Manager/Automating Deployments/terraform-template-box.md]: Hey Denes, I guess the links need to have all lowercase to work

**Pull request merged**